Demo/Sources/Players/PlaybackView.swift

```diff
@@ -185,7 +191,7 @@ private struct MainView: View {
                 VolumeButton(player: player)
             }
         }
-        .opacity(isUserInterfaceHidden ? 0 : 1)
+        .opacity(shouldHideInterface ? 0 : 1)
```

Comment:
The change is not animated. Maybe:

```suggestion
        .opacity(shouldHideInterface ? 0 : 1)
        .animation(.defaultLinear, value: shouldHideInterface)
```

Reply:
Should be fixed.

Reply:
Wonderful, thanks!